src/styles/components/navbar.sass

```diff
@@ -32,21 +32,29 @@ $navbar-breakpoint: $tablet
 $wp-max-width: 940px
 
 .navbar
+  font-size: 15px
+  font-weight: 600
+  display: flex
+  justify-content: space-between
+  align-items: center
   background-color: $navbar-background-color
   min-height: $navbar-height
   position: relative
   z-index: $navbar-z
   padding: $space-bigger
-  &.embedded
+
+.navbar.embedded
+  padding: 0.5rem 0 0.5rem 0.5rem
+  min-height: unset
+  +tablet()
     padding: 0.5rem 0
-    min-height: unset
-    +tablet()
-      max-width: $wp-max-width
-      margin-right: auto
-      margin-left: auto
+    max-width: $wp-max-width
+    margin-right: auto
+    margin-left: auto
 
 .navbar-brand
   align-items: stretch
+  justify-content: space
```

Comment:
Typo. Should it be `space-around` or `space-between`?

Reply:
apparently it isn't even necessary, so i'll simply remove the rule!